docs/configuration.asciidoc

```diff
@@ -250,7 +245,7 @@ minimized, but the agent will continue to poll the server for configuration chan
 | `ELASTIC_APM_GLOBAL_LABELS` |         | `dept=engineering,rack=number8`
 |============
 
-Labels added to all events, with the format key=value[,key=value[,...]].
+Labels added to all events with the format key=value[,key=value[,...]].
```

Comment:
This sentence is grammatically ambiguous. Does the format that you show relate to the events or the labels?

Reply:
```suggestion
Labels are added to all events. The format for labels is: `key=value[,key=value[,...]]`.
```